comixed-tasks/src/main/java/org/comixed/task/encoders/MoveComicTaskEncoder.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * ComiXed - A digital comic book library management application.
+ * Copyright (C) 2020, The ComiXed Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses>
+ */
+
+package org.comixed.task.encoders;
+
+import lombok.extern.log4j.Log4j2;
+import org.comixed.model.comic.Comic;
+import org.comixed.model.tasks.Task;
+import org.comixed.model.tasks.TaskType;
+import org.comixed.repositories.tasks.TaskRepository;
+import org.comixed.task.model.MoveComicWorkerTask;
+import org.springframework.beans.factory.ObjectFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+/**
+ * <code>MoveComicTaskEncoder</code> encodes instances of {@link MoveComicWorkerTask}.
+ *
+ * @author Darryl L. Pierce
+ */
+@Component
+@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+@Log4j2
+public class MoveComicTaskEncoder extends AbstractTaskEncoder<MoveComicWorkerTask> {
+  public static final String DIRECTORY = "destination-directory";
+  public static final String RENAMING_RULE = "renaming-rule";
+
+  @Autowired private ObjectFactory<MoveComicWorkerTask> moveComicWorkerTaskObjectFactory;
+  @Autowired private TaskRepository taskRepository;
+
+  private Comic comic;
+  private String directory;
+  private String renamingRule;
```

Comment:
Couldn't we use the lombok annotations?

Reply:
Yeah, let's open a new story to do that across the board.

Reply:
ok nice